<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTuneDialog</class>
 <widget class="QDialog" name="NewTuneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new tune</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="instrumentsLabel">
      <property name="text">
       <string>Instrument:</string>
      </property>
      <property name="buddy">
       <cstring>instrumentsComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="instrumentsComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Title:</string>
      </property>
      <property name="buddy">
       <cstring>titleLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="titleLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewTuneDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewTuneDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
